packages/cf-tables/src/cf-tables.less

```diff
@@ -35,6 +35,11 @@
 }
 
 .o-table {
+    caption {
```

Comment:
This should move to cf-core with other base table styles.

Reply:
Force pushed and updated in https://github.com/cfpb/capital-framework/pull/951/commits/0c4efd125be7c54f56d19eb868fb3b44a93b39c7

Reply:
Before:
<img width="530" alt="Screen Shot 2019-07-23 at 3 42 06 PM" src="https://user-images.githubusercontent.com/704760/61742522-4dc99400-ad61-11e9-9f45-619910ec86fb.png">

After:
<img width="543" alt="Screen Shot 2019-07-23 at 3 42 01 PM" src="https://user-images.githubusercontent.com/704760/61742527-51f5b180-ad61-11e9-81a2-808c3a56edd5.png">